### DISCUSSION TITLE
Connecting to a Forge Server using the Console Client

### DISCUSSION THREAD

**Discussion by SummerBoi** [ORelio/Minecraft-Console-Client#1931] [Q&A]
Is it possible to connect to a forge server? If yes, how? 
I'm attempting to connect to my own Forge server running all the mods 6, the account keeps getting kicked each time with the reason, "[ERROR] IOException: Unable to read data from the transport connection: An established connection was aborted by the software in your host machine.
[ERROR] An error occured when attempting to join this server."  on the console client and "....Unexpected custom data from client". 

I'm running Forge 1.16.5-36.2.26 on my server

**Comment by ORelio** [comment]
It's possible that one of your mods does not work with MCC, since MCC does not actually emulate the mods.

**Reply by SummerBoi** [reply]
Ah, probably that I guess, I'll see if I can try and make it work, thanks
